module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -362,16 +352,23 @@ protected function getItemStatusGroup($record, $callnumberSetting)
                 $callnumberSetting,
                 'Multiple Call Numbers'
             );
+
+            // Get combined availability for location
+            $locationStatus = $this->availabilityStatusManager->combine($details['items']);
+            $locationAvailability = $locationStatus['availability'];
+
             $locationInfo = [
-                'availability' => $details['available'] ?? false,
+                'availability' =>
```

Comment:
The test `VuFindTest/Mink/HoldingsTest.php` actually expects the "uncertain" availability to be handled like as "available". This could be fixed by using `$locationAvailability->isAvailable()` here instead.

But I thinks we could refactored this even more and also have the "uncertain" availability handled by its own. For example we could return `$locationAvailability->availabilityAsString()` and remove `status_unknown`. Then we can display the location's availability here https://github.com/vufind-org/vufind/blob/ad39f269eb57607b634b0fd83336a7db59004728/themes/bootstrap3/js/check_item_statuses.js#L62 depending on that string.

What do you think about that?